.github/workflows/test.yaml

```
@@ -16,5 +16,47 @@ jobs:
-      - name: Run tests
-        run: npm test
+      - name: Build ember app
+        run: npm ember build
+
+      - name: Upload built ember app
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist
+          path: dist
+
+  unitTests:
```

actionlint (1.7.4)

"runs-on" section is missing in job "unitTests"

(syntax-check)